tests/e2e/lib/cli_common.go

```
@@ -1,19 +1,31 @@
 	"os/exec"
 	"path/filepath"
 	"strings"
+	"time"
+)
+
+// Default timeout for CLI commands that may hang (e.g., log streaming)
+const DefaultCLITimeout = 5 * time.Minute
+
+// Default retry settings for CLI commands
+const (
+	DefaultCLIRetries     = 3
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)